database.py

```diff
@@ -45,17 +56,23 @@ def insert(self, obj):
                 self.__cur__.execute(query.insert_student, obj.to_db_entry())
                 self.__cur__.executemany(
                     query.insert_student_courses,
-                    [(obj.discord_id, course.lva_nr, course.semester) for course in obj.courses])
+                    [(obj.discord_id, course.lva_nr, course.semester, 1) for course in obj.courses])
```

Comment:
Someone used dark magic to get this 1 here... Whats going on? Should this be some kind of boolean? If yes, use True.